Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -150,40 +152,42 @@ class ActivityStreamPanel: UICollectionViewController, HomePanel {
 extension ActivityStreamPanel {
     enum Section: Int {
         case topSites
+        case pocketVideo
         case pocket
         case highlights
         case highlightIntro
 
-        static let count = 4
-        static let allValues = [topSites, pocket, highlights, highlightIntro]
+        static let count = 5
+        static let allValues = [topSites, pocketVideo, pocket, highlights, highlightIntro]
 
         var title: String? {
             switch self {
             case .highlights: return Strings.ASHighlightsTitle
             case .pocket: return Strings.ASPocketTitle
+            case .pocketVideo: return nil
```

Comment:
no title?

Reply:
I reuse the title from the .pocket case. `view.title = Section(Section.pocket.rawValue).title` Its probably simpler to just dupe the string here actually.